python/LAMBDA_DLQ_CHECK/parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Version": "1.0", 
+  "Parameters": {
+    "CodeKey": "LAMBDA_DLQ_CHECK.zip", 
+    "SourceRuntime": "python3.6", 
+    "RuleName": "LAMBDA_DLQ_CHECK", 
+    "SourceEvents": "AWS::Lambda::Function", 
+    "OptionalParameters": "{}", 
```

Comment:
Add the input parameter with an empty value. The RDK will not pass it, but it is a good way to document parameters.